stack/dashboard/rpm/wazuh-dashboard.spec

```diff
@@ -43,11 +43,7 @@ Wazuh dashboard is a user interface and visualization tool for security-related
 %prep
 
 # Set up required files
-if [ "%{_base}" = "s3" ];then
-    curl -kOL https://packages-dev.wazuh.com/stack/dashboard/base/%{DASHBOARD_FILE}
-else
-    cp /root/output/%{DASHBOARD_FILE} ./
-fi
+cp /opt/%{DASHBOARD_FILE} ./ ;\
```

Comment:
Which it's function? `;\`

Reply:
The base source directory change is due to the fact that the /root directory is a directory mounted at the boot of the container, for this reason, everything that exists in the image in the /root directory is replaced by the contents of the directory mounted.

For this reason, I decided to place the base in another dir and continue the installation with this reference